semeio/jobs/rft/gendata_rft.py

```diff
@@ -80,6 +88,12 @@ def run(well_times, trajectories, ecl_grid, ecl_rft, zonemap=None):
 
             _write_points(trajectory_points, fname)
             _write_inactive_info(trajectory_points, fname, zonemap)
+            dframes.append(
+                trajectory_points.to_dataframe().assign(well=well, time=time)
+            )
+
+    if csvfile is not None:
```

Comment:
Currently this doesn´t make that much sense as it is quite hard for a user to opt-out of this, as both the function and `argparse` default to something different than None.